src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -127,119 +126,235 @@ private function getPath(string $resourceShortName, array $operation, bool $coll
      *
      * @return \ArrayObject
      */
-    private function getPathOperation(string $operationName, array $operation, string $method, bool $collection, ResourceMetadata $resourceMetadata, array $mimeTypes) : \ArrayObject
+    private function getPathOperation(string $operationName, array $operation, string $method, bool $collection, string $resourceClass, ResourceMetadata $resourceMetadata, array $mimeTypes, \ArrayObject $definitions) : \ArrayObject
     {
         $pathOperation = new \ArrayObject($operation['swagger_context'] ?? []);
         $resourceShortName = $resourceMetadata->getShortName();
         $pathOperation['tags'] ?? $pathOperation['tags'] = [$resourceShortName];
         $pathOperation['operationId'] ?? $pathOperation['operationId'] = sprintf('%s%s%s', lcfirst($operationName), ucfirst($resourceShortName), ucfirst($collection ? 'collection' : 'item'));
 
-        if ('GET' === $method) {
-            $pathOperation['produces'] ?? $pathOperation['produces'] = $mimeTypes;
-
-            if ($collection) {
-                $pathOperation['summary'] ?? $pathOperation['summary'] = sprintf('Retrieves the collection of %s resources.', $resourceShortName);
-                $pathOperation['responses'] ?? $pathOperation['responses'] = [
-                    '200' => [
-                        'description' => sprintf('%s collection response', $resourceShortName),
-                        'schema' => [
-                            'type' => 'array',
-                            'items' => ['$ref' => sprintf('#/definitions/%s', $resourceShortName)],
-                        ],
-                    ],
-                ];
+        return $this->getPathOperationByMethod($method, $pathOperation, $mimeTypes, $collection, $resourceMetadata, $resourceClass, $resourceShortName, $operationName, $definitions) ?? $pathOperation;
```

Comment:
Maybe can be throw an error if the HTTP verb is unknown. IIUC its mandatory in the Swagger spec to use an existing HTTP verb: http://swagger.io/specification/#pathItemObject